electricity/views.py

```diff
@@ -1,12 +1,11 @@
-from django.contrib.auth.models import User
 from django.contrib.auth import authenticate, login, logout
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import HttpResponse, redirect, render
 from django.views.decorators.csrf import csrf_exempt
-from django.contrib import messages
-from django.contrib.auth.hashers import make_password, check_password
-from django.contrib.auth.decorators import login_required 
-from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.decorators import login_required
+from django.core.mail import send_mail
+
+from electricity.forms import UserForm
```

Comment:
Please pull the latest code from khannakshat7:master and then make a commit

Reply:
You are not working on the latest code

Reply:
Hello bhaiya,@khannakshat7 
Should I change all the commits again or its just for electricity/views,py file?

Reply:
Just for views.py file ig

Reply:
sorry, I am still not clear.
After cloning again the current code, I have to make the changes that I tried in all the files....right?

Reply:
So what issue are u facing?

Reply:
Please have a look on my new commit once

Reply:
Is it fine now?

Reply:
Please have a look on my new commit once
Is it fine now?

Reply:
Please have a look on this!